src/Workspaces/Core/Portable/FindSymbols/Declarations/DeclarationFinder_AllDeclarations.cs

```diff
@@ -93,19 +93,22 @@ internal static partial class DeclarationFinder
         private static async Task<(bool, ImmutableArray<SymbolAndProjectId>)> TryFindAllDeclarationsWithNormalQueryInRemoteProcessAsync(
             Project project, SearchQuery query, SymbolFilter criteria, CancellationToken cancellationToken)
         {
-            using (var session = await SymbolFinder.TryGetRemoteSessionAsync(
-                project.Solution, cancellationToken).ConfigureAwait(false))
+            if (RemoteSupportedLanguages.IsSupported(project.Language))
```

Comment:
Yuck - it only works in the local process because `Project.ContainsSymbolsWithNameAsync` return false for TS projects before we happen to get a `Compilation`.  We should probably move these checks higher up, and make it clearer that they don't work for TS/F#/Xaml in the API docs.